spec.html

```diff
@@ -28132,278 +28130,159 @@ <h1>Function Properties of the Math Object</h1>
       <emu-clause id="sec-math.abs">
         <h1>Math.abs ( _x_ )</h1>
         <p>Returns the absolute value of _x_; the result has the same magnitude as _x_ but has positive sign.</p>
-        <ul>
-          <li>
-            If _x_ is *NaN*, the result is *NaN*.
-          </li>
-          <li>
-            If _x_ is *-0*, the result is *+0*.
-          </li>
-          <li>
-            If _x_ is *-&infin;*, the result is *+&infin;*.
-          </li>
-        </ul>
+        <p>When the `Math.abs` method is called with argument _x_, the following steps are taken:</p>
+        <emu-alg>
+            1. Let _n_ be ? ToNumber(_x_).
+            1. If _n_ is *NaN*, return *NaN*.
+            1. If _n_ is *-0*, return *+0*.
+            1. If _n_ is *-&infin;*, return *+&infin;*.
+            1. If _n_ < 0, return -_n_.
+            1. Return _n_.
+        </emu-alg>
       </emu-clause>
 
       <emu-clause id="sec-math.acos">
         <h1>Math.acos ( _x_ )</h1>
         <p>Returns an implementation-approximated value representing the result of the arc cosine of _x_. The result is expressed in radians and ranges from *+0* to +&pi;.</p>
```

Comment:
If we're unifying on "inverse" instead of "arc" here and below, this paragraph should change too.